lib/rules/ruleWarningMessages.js

```diff
@@ -0,0 +1,23 @@
+import entries from 'object.entries';
```

Comment:
No, not really...
`W
  var configRuleEntries = Object.entries(configuration.rules);
                                 ^

TypeError: Object.entries is not a function`